### PR TITLE
update: auth plugin for oracle cloud infrastructure (OCI)

### PR DESCRIPTION
**Pull request by austingebauer**
Updates `go.mod` to bring in https://github.com/hashicorp/vault-plugin-auth-oci/commit/7f7ad0f784bc8c5058bd52228c45461bae1248b2.

The following steps were taken:
1. `go get github.com/hashicorp/vault-plugin-auth-oci@v0.5.5`
2. `go mod tidy`
3. `go mod vendor`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -80,7 +80,7 @@ require (
 	github.com/hashicorp/vault-plugin-auth-jwt v0.6.2
 	github.com/hashicorp/vault-plugin-auth-kerberos v0.1.5
 	github.com/hashicorp/vault-plugin-auth-kubernetes v0.6.1
-	github.com/hashicorp/vault-plugin-auth-oci v0.5.4
+	github.com/hashicorp/vault-plugin-auth-oci v0.5.3-beta1.0.20200611202039-7f7ad0f784bc
```
Comment (kalafut) [inline go.mod:83]: As we discussed, might be good to push a new tag to the OCI master branch so this doesn't look quite so odd?

File: api/go.sum [hunk 2]
```diff
@@ -8,12 +8,15 @@ github.com/aws/aws-sdk-go v1.25.37/go.mod h1:KmX6BPdI08NWTb3/sm4ZGu5ShLoqVDhKgpi
 github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973/go.mod h1:Dwedo/Wpr24TaqPxmxbtue+5NUziq4I4S80YR8gNf3Q=
 github.com/bgentry/speakeasy v0.1.0 h1:ByYyxL9InA1OWqxJqqp2A5pYHUrCiAL6K3J+LKSsQkY=
 github.com/bgentry/speakeasy v0.1.0/go.mod h1:+zsyZBPWlz7T6j88CTgSN5bM796AkVf0kBD4zp0CCIs=
+github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
```
Comment (kalafut) [inline api/go.sum:11]: Are all of these `go.mod` changes the result of a `tidy`, or does one need to be run?

File: go.mod (hunk 1 above)
Comment (austingebauer) [reply go.mod:83]: Ah, yes. The `beta1` showed up. I missed this. Will push a tag and update this PR.

Comment (austingebauer) [reply go.mod]: I've tagged vault-plugin-auth-oci [v0.5.5](https://github.com/hashicorp/vault-plugin-auth-oci/releases/tag/v0.5.5) brough that into `go.mod` now.

File: api/go.sum (hunk 2 above)
Comment (austingebauer) [reply api/go.sum:11]: These changes in `api/go.sum` appear after initially running `go get github.com/hashicorp/vault-plugin-auth-oci@v0.5.5`. Running `go mod tidy` subsequently doesn't remove these changes.

File: go.mod (hunk 1 above)
Comment (kalafut) [reply go.mod]: Thanks!

**Pull request merged**